### PR TITLE
Fix non-internal deps for split release (#19448)

### PR DESCRIPTION
**Pull request by samuel-williams-da**
Forward port of #19448

### CODE REVIEW

File: ci/prs.yml [hunk 1]
```diff
@@ -141,6 +141,7 @@ jobs:
       var_name: bash-lib
   - bash: |
       set -euo pipefail
+      cd sdk
```
Comment (dylant-da) [inline ci/prs.yml:144]: Why is this in this PR?

**Review by dylant-da — COMMENTED** [review]
Aside from the question, looks good, thanks!

**Review by dylant-da — APPROVED** [review]

File: ci/prs.yml (hunk 1 above)
Comment (samuel-williams-da) [reply ci/prs.yml:144]: Missing from a different PR, needed to make full compat work post moving into `sdk`

Comment (samuel-williams-da) [reply ci/prs.yml:144]: It was specifically fixed in this PR
https://github.com/digital-asset/daml/commit/89a57673b07ef2d7f7a4d400eb5758425c2170f0
Which was a 2.x specific fix for the compatibility tests.
Given 3.x doesn't run the same tests, this wasn't forward ported

**Pull request merged**